### ISSUE TITLE
There is no documentation

### ISSUE THREAD

**Issue by worikgh** [ApiGen/ApiGen#1019]
The docs directory is almost empty
apigen --help
apigen -h
apigen

None of the above output a usage summary.

**Comment by jordanbrauer** [comment]
What sort of docs/usage are you looking for?

Command usage summaries are not in the `docs/` directory that you mention – they are configured  from within the command itself.

If it's a set of documentation (as in a manual) you are looking for, I think the Usage section in the [README](https://github.com/ApiGen/ApiGen#usage) has all you need! 🙂 From my understanding of the tool so far, there isn't really much to document for usage other than a `src`/`dest` (I could be wrong though).

**Comment by worikgh** [comment]
ls /usr/share/doc/php-apigen
changelog.Debian.gz  copyright  UPGRADE-4.0.md

$ man php-apigen
No manual entry for php-apigen

This is from the Ubuntu 17.10 php-apigen package

**Comment by worikgh** [comment]
ls /usr/share/doc/php-apigen
changelog.Debian.gz  copyright  UPGRADE-4.0.md

$ man php-apigen
No manual entry for php-apigen

This is from the Ubuntu 17.10 php-apigen package

In the README.md at https://github.com/ApiGen/ApiGen there is no description of visibilityLevels for example.  What does it do?

**Comment by jordanbrauer** [comment]
I had no clue you can install this not via Composer. 😆 Are you using `v4.0` or `v5.0`? 

Try and install the program via composer instead of Ubuntu's `apt`.

> In the README.md at https://github.com/ApiGen/ApiGen there is no description of visibilityLevels for example. What does it do?

This refers to the visibility levels you want to allow in your documentation. `E.g.`, omitting `private` (as shown in the example) will omit all `private` properties & methods from the generated docs.

**Comment by worikgh** [comment]
I do not think this is the venu for explaining visibilityLevels etcetera.  The problem is with every aspect of the programme.  

A man page would be very useful.    But putting documentation into the project, documenting configuration and use so some one like me,. a very experienced computer programmer but with zero knowledge of your programme, can use it.  Then some one like me will write the man page for you.
Write for the person with zero knowledge of your programme, otherwise it is only used by people with close access to some one who knows it well and it becomes a burden on other projects to bring new programmers up to speed.

**Comment by jordanbrauer** [comment]
I understand and agree with everything you say. 👍 


What is the consensus on non-composer distribution of this project @vlastavesely?

**Comment by vlastavesely** [comment]
Consensus? Using composer is a preferred way to do things. The `APT` package in Ubuntu's repositories is just a remnant from the past.
